components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/BPELServerImpl.java

```diff
@@ -1009,11 +1037,52 @@ public boolean isLeader() {
             HazelcastInstance hazelcastInstance = BPELServiceComponent.getHazelcastInstance();
             Member leader = hazelcastInstance.getCluster().getMembers().iterator().next();
             if (leader.localMember()) {
+                log.debug("ODEClusterImpl#isLeader: true");
                 return true;
             }
+            log.debug("#ODEClusterImpl#isLeader: false");
```

Comment:
Wrap the statement with `if (log.isDebugEnabled())`

Reply:
Fixed in 349c0e4